conda_pip/main.py

```diff
@@ -110,13 +125,71 @@ def run_pip_install(
         command.append("-" + ("v" * verbosity))
     if force_reinstall:
         command.append("--force-reinstall")
-    if yes:
-        command.append("--yes")
     if upgrade:
         command.append("--upgrade")
     command.extend(specs)
 
     logger.info("pip install command: %s", command)
     process = run(command)
-    if process.returncode:
-        return process.returncode
+    return process.returncode
+
+
+def place_externally_managed(prefix: os.PathLike = None) -> Path:
+    """
+    conda-pip places its own EXTERNALLY-MANAGED file when it is installed in an environment.
+    We also need to place it in _new_ environments created by conda. We do this by implementing
+    some extra plugin hooks.
+    """
+    base_dir = get_env_stdlib(prefix or sys.prefix)
+    externally_managed = Path(base_dir, "EXTERNALLY-MANAGED")
+    if externally_managed.exists():
+        return
+    logger.info("Placing EXTERNALLY-MANAGED in %s", base_dir)
+    shutil.copy(HERE / "data" / "EXTERNALLY-MANAGED", externally_managed)
```

Comment:
This should use https://docs.python.org/3/library/importlib.resources.html instead of `__file__` for portability